fuzzbench/test_e2e_run.py

```diff
@@ -26,9 +28,21 @@
 class TestEndToEndRunResults:
     """Checks the result of a test experiment run."""
 
-    def test_all_jobs_finished_sucessfully(self):
-        """Fake test to be implemented later."""
-        assert True
+    def test_jobs_dependency(self):
+        """Tests that jobs dependency preserves during working."""
+        jobs = {
+            name: Job.fetch(name, connection=redis.Redis(host='queue-server'))
```

Comment:
nit: consider using a fixture for the redis_connection, ie something like:
```python
@pytest.fixture(scope="module")
def redis_connection():
  return redis.Redis(host='queue-server')
```
See https://docs.pytest.org/en/stable/fixture.html#fixture

Reply:
Sounds good! Let's put it at a separate PR to implement then.

Reply:
I don't see why it shouldn't be added in this PR.

Reply:
Sounds good too.